src/main/webapp/app/course/manage/detail/course-detail.component.html

```diff
@@ -204,6 +216,32 @@ <h2><span jhiTranslate="artemisApp.course.detail.title">Course Details:</span></
                     <span>{{ course.maxRequestMoreFeedbackTimeDays }}</span>
                 </dd>
             </ng-container>
+            <dt><span jhiTranslate="artemisApp.course.maxComplaintTextLimit.title">Maximum number of characters per complaint</span></dt>
+            <dd id="course-max-complaint-text-limit">
+                <span>{{ course.maxComplaintResponseTextLimit }}</span>
+            </dd>
+            <dt><span jhiTranslate="artemisApp.course.maxComplaintResponseTextLimit.title">Maximum number of characters per complaint response</span></dt>
+            <dd id="course-max-complaint-response-text-limit">
+                <span>{{ course.maxComplaintResponseTextLimit }}</span>
+            </dd>
```

Comment:
Probably only makes sense to show this in case complaints are enabled for the course.

I’d also group it together with the other complaint settings rather than having `maxRequestMoreFeedbackTimeDays` in between.